repositories/MerchStoreRepository.ts

```diff
@@ -80,11 +90,7 @@ export class MerchItemOptionRepository extends BaseRepository<MerchandiseItemOpt
     await qb
       .update()
       .set({ discountPercentage })
-      .where(`item IN ${qb.subQuery()
-        .select('merch.uuid')
-        .from(MerchandiseItemModel, 'merch')
-        .where('merch.collection = :collection')
-        .getQuery()}`)
+      .where('merch.collection = :collection')
```

Comment:
This subquery exists because item options don't have a `collection` property. Items have the `collection` property and item options have an `item` property. So, if we want to update all the item options in a collection, we need to get all the items in the collection first and then update all the item options where `item` is in that list of items. Another way of thinking about this is that we're doing a JOIN on items and item options and then filtering by `collection`.

You can verify this for yourself by writing a quick test that calls this method, and see what happens. The test should fail because we throw some sort of invalid SQL error.